README.md

```diff
@@ -1,73 +1,16 @@
 # Amsterdam React Maps
+A repo with components / utilities to be used in projects published by [Amsterdam](https://github.com/Amsterdam). 
+It also contains an "example" directory where we put so-called "recipes" that developers at Amsterdam can basically copy/paste in their own projects
 
-To develop (and use npm link to link ASC and react-maps), please read: https://reactjs.org/warnings/invalid-hook-call-warning.html#duplicate-react
+## Usage
 
+NOTE: we haven't released a package yet! This will be done soon
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+With npm: `npm install --save @datapunt/amsterdam-react-maps`
```

Comment:
The latest version on NPM has `--save` by default, but doesn't hurt to mention it explicitly.